device-types/Cisco/C9800-L-C.yaml

```diff
@@ -3,8 +3,12 @@ manufacturer: Cisco
 model: Catalyst 9800-L-C
 slug: cisco-c9800-l-c
 part_number: C9800-L-C
-u_height: 1
+u_height: 0
+subdevice_role: child
```

Comment:
Before you ask @danner26 this I think is perfectly acceptable.  The device can be slotted into a 1.5 (or 2, can't remember) RU shelf (2 WLC's in 1 device).

Reply:
The Cisco provided one is a 2u shelf.. we actually mount this on a 1u shelf ourselves. But I've added the official 2U shelf as part of this PR